### DISCUSSION TITLE
Seismic anistropy/property modelling

### DISCUSSION THREAD

**Discussion by Liujianhua00** [mtex-toolbox/mtex#1569] [Ask Anything] [answered]
Hello MTEX community,
When I use MTEX to calculate the seismic anistropy/property of rocks, I can carry out EBSD tests to get the original data and calculate the ODF and seismic properties at last. But if I want to do some modelling about seismic anistropy，how  can I get the ODF of the specific mineral? 

Thanks

**Comment by lfgmorales** [comment]
Hi,



I am not sure I understood your question, but you can either


1) use the ODFs you have and use those as input to your "modelling" or


2) generate a texture evolution numerically (e.g. VPSC modelling, Taylor - this one available in MTEX - https://mtex-toolbox.github.io/TaylorModel.html), calculate the ODF out of those, and use them as input for your seismic property modelling


3) use some of the pre-defined ODFs from MTEX (e.g. https://mtex-toolbox.github.io/UnimodalODFs.html, https://mtex-toolbox.github.io/FibreODFs.html), compatible with the CPOs you expect out of your phases, and use those as input


Note however that 1 and 2 are the more realistic options, option 3 will only have some physical meaning for a very limited number of phases



cheers



Luiz





Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
HPM C57.3

Otto-Stern-Weg 3

8093 Zürich

Switzerland

Phone: + 41 44 633 3746

<tel:+41446333746>E-mail: ***@***.***



"No trees were destroyed in the sending of this message, however, a significant number of electrons were terribly inconvenienced"


________________________________
From: Liujianhua00 ***@***.***>
Sent: 23 September 2022 13:35:43
To: mtex-toolbox/mtex
Cc: Subscribed
Subject: [mtex-toolbox/mtex] Seismic anistropy/property modelling (Discussion #1569)


Hello MTEX community,
When I use MTEX to calculate the seismic anistropy/property of rocks, I can carry out EBSD tests to get the original data and calculate the ODF and seismic properties at last. But if I want to do some modelling about seismic anistropy，how can I get the ODF of the specific mineral?

Thanks

―
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1569>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN7WHJZBNVGPHXVCNH3V7WIY7ANCNFSM6AAAAAAQT4X7RU>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1569", "url": "https://github.com/mtex-toolbox/mtex/discussions/1569", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Liujianhua00** [reply]
Thank you very much for your kind suggestions and I will give it a try to them. In fact, I have tried to use BinghamODF to model the fabric, but the result displays a fabric more like a single crystal. Maybe there is something with it.
Here I also explain more accurately of my aim.  The fabric strength and fabric type of minerals (e.g. amphibole and olivine) can affect the seismic anistropy greatly. However, the fabric strength and fabric type of constituted mineral in deformed rocks are determined, so the problem is how to quantitatively evaluate the effect of fabric strength and the successive transition of fabric type? Thus, I consider to model the different fabrics and during this modelling process I can also control the fabric strength. Furtherly, I can calculate the ODF and the seismic anistropy.

**Comment by lfgmorales** [comment, marked as answer]
Hi

In this case you should do numerical modelling of CPO evolution, then use for example some of the CPOs generated at different strain steps to calculate Your OdFs and derive both the fabric strength and the seismic properties.

For olivine this was already done, for example, Andrea Tommasi paper of 2000

https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JB900411

There is also the work of Rudy Wenk, Yuval Boneh, Lars Hansen…the list is immense. For amphibole there is less work,

Hope this helps

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 23 Sep 2022, at 14:43, Liujianhua00 ***@***.***> wrote:

﻿

Thank you very much for your kind suggestions and I will give it a try to them. In fact, I have tried to use BinghamODF to model the fabric, but the result displays a fabric more like a single crystal. Maybe there is something with it.
Here I also explain more accurately of my aim. The fabric strength and fabric type of minerals (e.g. amphibole and olivine) can affect the seismic anistropy greatly. However, the fabric strength and fabric type of constituted mineral in deformed rocks are determined, so the problem is how to quantitatively evaluate the effect of fabric strength and the successive transition of fabric type? Thus, I consider to model the different fabrics and during this modelling process I can also control the fabric strength. Furtherly, I can calculate the ODF and the seismic anistropy.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1569#discussioncomment-3717070>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6TBZ7ASUSPXIYBQNDV7WQWRANCNFSM6AAAAAAQT4X7RU>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1569#discussioncomment-3717070", "url": "https://github.com/mtex-toolbox/mtex/discussions/1569#discussioncomment-3717070", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Liujianhua00** [reply]
Lucky day, it's a very nice suggestion. I'm trying to carry out some modelling of amphibole but miss the numerous work have already been done on olivine. It will provide a reference to modelling in amphibole.
Thanks again.

**Comment by lfgmorales** [comment]
You may want to look at the work of Dan Tatham

https://www.sciencedirect.com/science/article/abs/pii/S0012821X07007650

Feom the Jung‘s group

https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2019GL085189

From Ji‘s group

https://d1wqtxts1xzle7.cloudfront.net/44885458/A_new_calibration_of_seismic_velocities_20160419-18423-wknrj3-with-cover-page-v2.pdf?Expires=1664004348&Signature=UW57lBxVmnDp2gPG7-70JPgOtbupqxM7aYgjVp5KTRwwRzgN4Z1mjJv8eQOnCW0hKczIIJuxfQ-RBbjmStN1zw9xafMNciNFnu0yenVPhAm7kmCeGz3NOHbQSXOT8VC1F4l6UvOUc8eTHv-LRt-5MWoG4Uw-MS-f~3ldZa3dlGl0QiJhjteNUJ8xsQSuVOLogP5aJ3B6xyb-OgvzSxlm1HgAMArs41nn7odrNrHnnFoXPfVF-BpDXCYR2ZCBn5aeKw8BFabBnYmvRvMVf7~0~YV7c900IYcDdHS4O52DYCrfJbliZ8c6PYCCQeqY7S9HE2V3cKcyuusD5FuWTXBLag__&Key-Pair-Id=APKAJLOHF5GGSLRBV4ZA

Just google for it Abd there will be some papers

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 24 Sep 2022, at 02:52, Liujianhua00 ***@***.***> wrote:

﻿

Lucky day, it's a very nice suggestion. I'm trying to carry out some modelling of amphibole but miss the numerous work have already been done on olivine. It will provide a reference to modelling in amphibole.
Thanks again.

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1569#discussioncomment-3721207>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5INZ4NGB2L4B7PCXNDMTV7ZGEVANCNFSM6AAAAAAQT4X7RU>.
You are receiving this because you commented.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1569#discussioncomment-3721207", "url": "https://github.com/mtex-toolbox/mtex/discussions/1569#discussioncomment-3721207", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Liujianhua00** [reply]
Thanks!
